### PR TITLE
Refs [TASK][34036] Create UI Home Screen

### PR DESCRIPTION
**Pull request by phucsaker99**
## Related Tickets
- [#Task 34036: Create UI Home Screen](https://edu-redmine.sun-asterisk.vn/issues/34036)
## WHAT
- Create TextView
- Create option menu (language + notification)
- Create recyclerview result api by category, ingredients 
## Evidence (Screenshot or Video)
image
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -1,6 +1,118 @@
...
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/textYummy"
+            android:layout_width="wrap_content"
+            android:layout_height="39dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="8dp"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_home.xml]: hard code

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -1,6 +1,118 @@
...
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <EditText
+            android:id="@+id/editSearchMeal"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_40"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginBottom="8dp"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_home.xml]: tương tự

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -1,6 +1,118 @@
...
+            android:layout_marginTop="@dimen/dp_15"
+            android:text="@string/title_category"
+            android:textColor="@color/red_berry"
+            android:textSize="@dimen/sp_20"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/view" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerView"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
            android:id="@+id/recyclerABC"
```
viết cụ thể hơn chút

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -22,4 +22,6 @@
     <dimen name="dp_150">150dp</dimen>
     <dimen name="dp_200">200dp</dimen>
     <dimen name="dp_400">400dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
```
Comment (thonc-2135) [inline app/src/main/res/values/dimens.xml]: sắp xếp theo thứ tự

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -1,6 +1,115 @@
...
+            android:textColor="@color/red_berry"
+            android:textSize="@dimen/sp_20"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/view" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerCategory"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_300"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_home.xml]: sao fix cứng size to thế, em đo theo tỉ lệ màn hình thôi

**Pull request merged**